### ISSUE TITLE
New event dashboard order list

### ISSUE THREAD

**Issue by Jasonvdb** [big-neon/bn-web#1523]
https://share.goabstract.com/eb756c85-a087-4208-b3d3-ac4d5ca002a9?collectionLayerId=e6f5fc83-c651-48b7-be93-3db8123c34b3&mode=design

**Comment by Jasonvdb** [comment]
@Tas4tari do we have mobile designs for this page?

**Comment by Tas4tari** [comment]
@Jasonvdb I have requested the mobile view for this page from the designers, just awaiting feedback.

**Comment by Tas4tari** [comment]
https://share.goabstract.com/f3168080-6095-4805-85c8-6e7b29505333

**Comment by Jasonvdb** [comment]
Mobile design: https://app.goabstract.com/share/f3168080-6095-4805-85c8-6e7b29505333

**Comment by Anziskavds** [comment]
Tested on develop and is working as expected. 

Only the notes doesn't show yet after adding as per @Jasonvdb comment on the issue.
https://cl.ly/78fade8634ed

All else works as expected on develop.

**Comment by Jasonvdb** [comment]
Yes comment here: https://github.com/big-neon/bn-web/issues/1515#issuecomment-514125352

**Comment by Anziskavds** [comment]
Tested successful on Beta on web and mobile web.
